Project.toml

```diff
@@ -3,14 +3,12 @@ uuid = "c4c386cf-5103-5370-be45-f3a111cca3b8"
 version = "0.3.0"
 
 [deps]
-BinDeps = "9e28174c-4ba2-5203-b857-d8d62c4213ee"
 Cairo = "159f3aea-2a34-519c-b102-8c37f9878175"
-Homebrew = "d9be37ee-ecc9-5288-90f1-b9ca67657a75"
-Libdl = "8f399da3-3557-5675-b5ff-fb832c97cbdb"
-WinRPM = "c17dfb99-b4f7-5aad-8812-456da1ad7187"
+Glib_jll = "7746bdde-850d-59dc-9ae8-88ece973131d"
+Librsvg_jll = "925c91fb-5dd6-59dd-8e8c-345e74382d89"
 
 [compat]
-julia = "0.7, 1"
+julia = "≥ 1.3"
```

Comment:
This should just be `julia="1.3"`, otherwise you are declaring compat with Jualia 2, which I think is not best practice right now.